openlibrary/plugins/upstream/utils.py

```diff
@@ -1222,13 +1222,16 @@ def _get_author_config():
 
     The results are cached on the first invocation.
     Any changes to /config/author page require restarting the app.
-
     """
-    thing = web.ctx.site.get('/config/author')
-    if hasattr(thing, "identifiers"):
-        identifiers = [Storage(t.dict()) for t in thing.identifiers if 'name' in t]
-    else:
-        identifiers = {}
+    # Load the author config from the author.yml file in the author directory
```

Comment:
We’re only loading `identifiers` of the author config here… but that’s also currently the entirety of the config. I don’t know if that’s something that needs changing or not.